app/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
 
     defaultConfig {
         applicationId "infrmr.newsapp.github.com.ifrmr"
-        minSdkVersion 19
+        minSdkVersion 15
```

Comment:
We should start out trying to support 8 and up. We can move the min version up if we come across a dependency that needs a certain sdk, but the goal should be to support as many devices as possible.